api/src/main/kotlin/com/openlattice/search/Auth0UserSearchFields.kt

```diff
@@ -0,0 +1,20 @@
+package com.openlattice.search
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+// https://auth0.com/docs/users/user-search/user-search-query-syntax
+data class Auth0UserSearchFields(
+    @JsonProperty val email: String?,
+    @JsonProperty val name: String?
```

Comment:
so you probably do still want the default values here so that users can pass
```
{
  "email": "blah"
}
```
instead of 
```
{
  "email": "blah",
  "name": null
}
```
(or vice versa)

Reply:
Passing `null` for `name` is not necessary. This works as expected:

```
{
  "email": "blah"
}
```

Reply:
cool cool. ah the other thing is that you don't need to pass `@JsonProperty` -- Jackson knows how to properly serialize/deserialize data classes

Reply:
Updated.